README.md

```diff
@@ -1 +1,21 @@
 # LFP Appeals Processor API
+
+## How to run
+### Run as a Spring Boot application
+`mvn clean install spring-boot:run`
+
+## Useful Endpoints
+### Health
+http://localhost:9000/actuator/health
+
+### API Documentation (Swagger)
+http://localhost:9000/swagger-ui.html
+
+## Technologies
```

Comment:
I suggest moving it to top of the page as first paragraph so it gives bit of context before more hand-on content telling how to run and test an app.